src/components/duration/_macro.njk

```diff
@@ -23,7 +23,7 @@
         {% if params.field1 is defined and params.field1 %}
             {{ onsInput({
                 "id": params.field1.id,
-                "classes": "input--w-2" + (" js-exclusive-group-item" if params.mutuallyExclusive else ""),
+                "classes": "input--w-2" + (" input--error" if params.field1.error else "") + (" js-exclusive-group-item" if params.mutuallyExclusive else ""),
```

Comment:
I think this should require both `params.error` and `params.field1.error`. Otherwise a user to pass error into the field but not provide the error details.
```suggestion
                "classes": "input--w-2" + (" input--error" if params.error and params.field1.error else "") + (" js-exclusive-group-item" if params.mutuallyExclusive else ""),
```
doesn't it also need `is defined`?

Reply:
Actually, would it make more sense to make the user pass something like `params.field1.noError` to remove the class and styling, so it's only something additional to add one of the fields in the case that there's only a single error.  Otherwise, users may not end up using this new param because they have errors in both inputs, and we'd lose the border styling.

Reply:
Ive sorted this, I decided to change the logic so that:
- if just `params.error` is passed then both will have the error style
- if `params.error` and one of the fields has `error` passed then that one will have error style added but the other one wont
- if `params.error` and both of the fields has `error` passed then both fields get the error style

I thought this was the best way forward as it made more sense to me from a dev point of view than adding a `noError` param to remove the style